objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -660,4 +743,22 @@ boolean valEquals(Object actual, Object expected) {
 
         return actual.equals(expected);
     }
+    
+    ExtensionServiceMessage createRealConfig() {
+        Map msg = new LinkedHashMap();
+        Map object = new LinkedHashMap();
+        Map neuralNet = new LinkedHashMap();
+        
+        neuralNet.put("pbFile", PB_FILE);
+        neuralNet.put("labelFile", LABEL_FILE);
+        neuralNet.put("outputDir", OUTPUT_DIR);
+        neuralNet.put("saveRate", SAVE_RATE);
+        neuralNet.put("anchors", Arrays.asList(1,1,1,1,1,1,1,1,1,1));
```

Comment:
Should either have a String JSON message that we suck in via a JSON converter or something.  I want to make certain that we show what JSON we expect.